packages/react/select/src/Select.tsx

```diff
@@ -196,8 +201,9 @@ interface SelectTriggerProps extends PrimitiveButtonProps {}
 
 const SelectTrigger = React.forwardRef<SelectTriggerElement, SelectTriggerProps>(
   (props: ScopedProps<SelectTriggerProps>, forwardedRef) => {
-    const { __scopeSelect, disabled = false, ...triggerProps } = props;
+    const { __scopeSelect, ...triggerProps } = props;
     const context = useSelectContext(TRIGGER_NAME, __scopeSelect);
+    const disabled = props.disabled || context.disabled;
```

Comment:
We should still pull out the prop otherwise the trigger prop can end up overriding the value from context which wouldn't make sense.

We generally do this like:

```tsx
const { __scopeSelect, disabled: disabledProp = false, ...triggerProps } = props;

…

const disabled = disabledProp || context.disabled;
```

Reply:
Thanks for the review, I just did what you suggested